docs/CONFIGURING_TASKPROV.md

```diff
@@ -0,0 +1,144 @@
+# Taskprov Extension
+
+Janus has limited support for the Taskprov extension, as defined in
+[draft-wang-ppm-dap-taskprov][1]. We support it largely to facilitiate
+integrations with Divvi Up partners who have requested support for the extension.
+
+Taskprov defines a mechanism for provisioning tasks into a DAP aggregator. Task
+configuration is performed in-band as clients submit reports. Clients will submit
+reports containing the task configuration, and the aggregator will respond by
+validating and provisioning the task. The exact mechanism of how the task
+configuration is provided depends on the version of the implemented spec draft.
+
+[1]: https://datatracker.ietf.org/doc/draft-wang-ppm-dap-taskprov/
+
+## Compatibility
+
+See this table for the matrix of branches to Taskprov versions. Taskprov is not
+implemented on any branches missing from this list.
+
+| Git branch | Taskprov draft version | Conforms to protocol? | Status |
+| ---------- | ------------- | --------------------- | ------ |
+| `release/0.subscriber-01` | [`draft-wang-ppm-dap-taskprov-02`][2] | Helper only | Supported |
```

Comment:
I think we don't currently have a taskprov implementation merged into `release/0.subscriber-01`, so if there's an issue open for that, you could link it here.

Reply:
It's hanging out in https://github.com/divviup/janus/pull/1742, I got ahead of myself a bit in this doc to spare having to change it for a change that's <1 day from being applied.